Maple2.Server.Game/Model/Field/Actor/ActorState/AnimationState.cs

```diff
@@ -0,0 +1,220 @@
+﻿using Maple2.Model.Metadata;
+using Maple2.Server.Core.Packets;
+using Maple2.Server.Game.Model.Enum;
+
+namespace Maple2.Server.Game.Model.Field.Actor.ActorState;
+
+public class AnimationState {
+    // TODO: maybe make this a ping based variable?
+    private const long ClientGraceTimeTick = 500; // max time to allow client to go past loop & sequence end
```

Comment:
should this go into constants?

Reply:
Yeah that sounds like a good place for it